src/interface/storage.thrift

```diff
@@ -163,6 +168,24 @@ struct AddEdgesRequest {
     3: bool overwritable,
 }
 
+struct EdgeKeyRequest {
```

Comment:
Why not use go sentence

Reply:
`Go` can only return specified `edgeType`  edgeKeys of a vertex.

Reply:
I see.